src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,273 @@
+import { cva, VariantProps } from "class-variance-authority";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const shellStylesOuter = cva(
+  "fixed inset-0 mx-auto mx-auto flex w-full max-w-4xl justify-center rounded-lg px-4 px-4 sm:px-6 sm:px-6 md:px-8 md:px-8",
+  {
+    variants: {
+      intent: {
+        primary:
+          "border-solid border-2 border-zinc-200 bg-white dark:bg-zinc-900",
```

Comment:
For the future, anything that's shared between all variants can be yanked up to the "invariant" string on L6